src/command_modules/azure-cli-configure/azure/cli/command_modules/configure/__init__.py

```diff
@@ -0,0 +1,111 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+from __future__ import print_function
+import os
+from six.moves import input, configparser #pylint: disable=redefined-builtin
+from azure.cli.commands import cli_command
+import azure.cli._logging as _logging
+from azure.cli._config import GLOBAL_CONFIG_PATH, ACTIVE_ENV_CONFIG_PATH, ENV_VAR_PREFIX
+import azure.cli.command_modules.configure._help # pylint: disable=unused-import
+
+logger = _logging.get_az_logger(__name__)
+
+OUTPUT_LIST = [
+    {'name': 'json', 'desc': 'JSON formatted output that most closely matches API responses'},
+    {'name': 'jsonc', 'desc': 'Colored JSON formatted output that most closely matches API responses'}, #pylint: disable=line-too-long
+    {'name': 'table', 'desc': 'Human-readable output format that focuses on clarity'},
+    {'name': 'tsv', 'desc': 'Tab and Newline delimited, great for GREP, AWK, etc.'}
+]
+
+MESSAGES = {
+    'intro': '\nWelcome to the Azure CLI! This command will guide you through logging in and '\
+             'setting some default values.\n',
+    'global_settings_loc': 'Your global settings can be found at {}',
+    'active_env_settings_loc': 'Your current environment settings can be found at {}',
+    'current_config_info': 'Your current configuration is as follows:\n',
+    'env_vars_heading': 'Environment variables:',
+    'global_settings_exists': 'Do you wish to change your global settings? (y/N): ',
+    'prompt_global_output': 'What default output format would you like?\n'\
+                            '{}\n'\
+                            'Please enter a choice [{}]: ',
+    'closing': '\nYou\'re all set! Here are some commands to try:\n'\
+               ' $ az vm create --help\n'\
+               ' $ az feedback\n',
+}
```

Comment:
These could just as easily be string constants. Then you get intellisense and Pylint's helpful (?) feedback. As is, if you misspell something you won't know until you run it and it crashes. Not as bad as the infamous "resource_id(**kwargs)" but still annoying. :)  Or you can leave it.